open_prices/api/proofs/serializers.py

```diff
@@ -49,3 +49,16 @@ class ProofUpdateSerializer(serializers.ModelSerializer):
     class Meta:
         model = Proof
         fields = Proof.UPDATE_FIELDS
+
+
+class ProofProcessWithGeminiSerializer(serializers.ModelSerializer):
```

Comment:
if nothing to do with the Proof serializer, you can simply rename to 
```suggestion
class ProofProcessWithGeminiSerializer(serializers.Serializer):
```

Reply:
https://www.django-rest-framework.org/api-guide/serializers/

Reply:
and remove the Meta